CondFormats/L1TObjects/interface/L1TMuonBarrelParams.h

```diff
@@ -7,12 +7,13 @@
 /// \author: Giannis Flouris
 ///
 
-#ifndef L1TBMTFParams_h
-#define L1TBMTFParams_h
+#ifndef CondFormats_L1TObjects_L1TMuonBarrelParams_h
+#define CondFormats_L1TObjects_L1TMuonBarrelParams_h
 
 #include <memory>
 #include <iostream>
 #include <vector>
+#include <map>
```

Comment:
@Dr15Jones do I understand correctly that this additional `include` statement is the actual fix, while the rest (namely `insert` -> `emplace`) is just performance improvement?